src/coreclr/interpreter/compiler.cpp

```diff
@@ -2586,26 +2633,44 @@ void InterpCompiler::EmitCall(CORINFO_RESOLVED_TOKEN* constrainedClass, bool rea
             break;
 
         case CORINFO_CALL_CODE_POINTER:
+        {
             if (callInfo.nullInstanceCheck)
             {
                 // If the call is a normal call, we need to check for null instance
                 // before the call.
                 // TODO: Add null checking behavior somewhere here!
             }
-            assert(!"Need to support calling a code pointer");
-            break;
 
+            EmitPushCORINFO_LOOKUP(callInfo.codePointerLookup);
+            m_pStackPointer--;
+            int codePointerLookupResult = m_pStackPointer[0].var;
+
+            calliCookie = m_compHnd->GetCookieForInterpreterCalliSig(&callInfo.sig);
+
+            AddIns(INTOP_CALLI);
+            m_pLastNewIns->data[0] = GetDataItemIndex(calliCookie);
+            m_pLastNewIns->SetSVars2(CALL_ARGS_SVAR, codePointerLookupResult);
+            break;
+        }
         case CORINFO_VIRTUALCALL_VTABLE:
             // Traditional virtual call. In theory we could optimize this to using the vtable
             AddIns(INTOP_CALLVIRT);
             m_pLastNewIns->data[0] = GetDataItemIndex(callInfo.hMethod);
             break;
 
         case CORINFO_VIRTUALCALL_LDVIRTFTN:
-            if (callInfo.exactContextNeedsRuntimeLookup)
+            if (callInfo.sig.sigInst.methInstCount != 0)
             {
-                // Resolve a virtual call using the helper function to a function pointer, and then call through that
-                assert(!"Need to support ldvirtftn path");
+                assert(extraParamArgLocation == INT_MAX);
```

Comment:
Should we have a comment explaining what it means if this assert fails? I assume it's for methods that have a hidden extra parameter as part of their calling convention, and we don't support those yet